releasenotes/2.1.1.md

```diff
@@ -0,0 +1,69 @@
+
+## <a id="highlights"></a>Release Highlights
+
+Apache jclouds 2.1.1 new features include:
+
+* Add support for c5 and R4 instance types in AWS ec2.
+* Add the new Paris region to aws-ec2
+* Support different "domain" for "identity/user" and "project" in OpenStack Keystone V3
+* Improve Azure ARM clean resources up especially for orphaned networks and orphaned resource groups.
+
+### Bugs and patches
+
+* [JCLOUDS-1200] - org.jclouds.http.HttpResponseException fails to serialize
+* [JCLOUDS-1263] - Listing objects in Openstack Swift blob store containers fails if a key has an invalid URL encode sequence
+* [JCLOUDS-1294] - Azure ARM improve HTTP retry logic
+* [JCLOUDS-1330] - Azure ARM orphaned networks not cleaned up when node deleted
+* [JCLOUDS-1331] - Azure ARM orphaned resource groups are not cleaned up due to api caching/timing issue
+* [JCLOUDS-1362] - Randomly generated node password can be invalid in Azure ARM
+* [JCLOUDS-1368] - GCS cannot upload multipart file larger than 1 GB
+* [JCLOUDS-1373] - Add the new Paris region to aws-ec2
+* [JCLOUDS-1381] - NullPointerException provisioning VM in GCE: could not find image for disk
+* [JCLOUDS-1389] - Multipart upload to Google cloud storage failing with 400 bad request
+* [JCLOUDS-1391] - Headers must be sorted by lowercase when calculating signature
+* [JCLOUDS-1392] - When fetching blob metadata with details all folders are removed from response
+* [JCLOUDS-1400] - GCS's presigned url requires blank Content-Type
+* [JCLOUDS-1401] - S3 SignatureDoesNotMatch when listing keys using prefix with ampersand
+* [JCLOUDS-1405] - Dimension Data Live Tests Broken - Domain classes for Server and Image with List nics incorrect
+* [JCLOUDS-1410] - SoftLayer cannot provision with private image
+* [JCLOUDS-1422] - LocalBlobStore.list ignores recursive flag when prefix set
+* [JCLOUDS-1427] - NPE when deserializing Elastic IPs that have tags
+* [JCLOUDS-1433] - IllegalArgumentException from Strings2.replaceTokens
+
+The complete list of fixed issues and improvements can be found [here](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12314430&version=12342884).
+
+## <a id="knownissues"></a> Known Issues
+
+There are a number of open JIRA issues related to OpenStack endpoint and API version support: [JCLOUDS-114](https://issues.apache.org/jira/browse/JCLOUDS-114), [JCLOUDS-773](https://issues.apache.org/jira/browse/JCLOUDS-773) and [JCLOUDS-1197](https://issues.apache.org/jira/browse/JCLOUDS-1197). In particular, jclouds does not currently support OpenStack Keystone v3; the community is currently [actively working on that](https://s.apache.org/OSbY).
+
+## <a id="credits"></a>Credits
+
+jclouds would like to thank everyone who contributed time and effort in order to make this release happen:
+
+* [Daniel Estevez](https://github.com/danielestevez) for his contributions to the Azure ARM provider.
```

Comment:
Not sure how this section works but should it include anyone who contributed to the release?

Reply:
We just try to mention who contributed. It's a nice way to say thanks and to let people feel more engaged.

Reply:
Then we should add https://github.com/jclouds/jclouds/issues?q=is%3Apr+author%3AVRanga000 for the EC2 new instance types and https://github.com/jclouds/jclouds/issues?q=is%3Apr+author%3Ajglick for the IllegalArgumentException fix in jclouds-core